webpack/assets/javascripts/react_app/components/SettingsTable/SettingsTableHelpers.js

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+
+import { Tooltip, OverlayTrigger } from 'patternfly-react';
+
+import { translate as __ } from '../../common/I18n';
+
+import { deepPropsToCamelCase } from '../../common/helpers';
+
+export const withTooltip = Component => componentProps => {
+  const { tooltipId, tooltipText, ...rest } = componentProps;
+
+  return (
+    <OverlayTrigger
+      overlay={<Tooltip id={tooltipId}>{tooltipText}</Tooltip>}
+      trigger={['hover', 'focus']}
+      placement="top"
+      rootClose={false}
+    >
+      <span>
+        <Component {...rest} />
+      </span>
+    </OverlayTrigger>
+  );
+};
+
+export const arraySelection = setting => {
+  const { selectValues } = setting;
+
+  if (!Array.isArray(selectValues)) {
+    return null;
+  }
+  return deepPropsToCamelCase(selectValues);
+};
+
+const formatEncryptedDefault = setting => {
+  if (setting.encrypted && setting.default) {
+    return setting.default
+      .split('')
+      .map(item => '\u2219')
+      .join('');
+  }
+
+  return null;
+};
+
+const formatBooleanDefault = setting => formatBoolean('default', setting);
+const formatBooleanValue = setting => formatBoolean('value', setting);
+
+const formatBoolean = (attr, setting) => {
+  if (setting.settingsType === 'boolean') {
+    if (setting[attr]) {
+      return __('Yes');
+    }
+    return __('No');
+  }
+  return null;
+};
+
+const formatArrayValue = setting => formatArray('value', setting);
+const formatArrayDefault = setting => formatArray('default', setting);
+
+const formatArray = (attr, setting) => {
+  if (setting.settingsType === 'array') {
+    return `[${
+      setting[attr] && setting[attr].length > 0 ? setting.value.join(', ') : ''
+    }]`;
+  }
+  return null;
+};
+
+const formatTextValue = setting => setting.value;
+const formatTextDefault = setting => setting.default;
+
+const formatEmpty = (attr, emptyValue, setting) => {
+  if (!setting[attr]) {
+    return emptyValue;
+  }
+  return null;
+};
+
+const formatEmptyDefault = setting =>
+  formatEmpty('default', __('Not set'), setting);
+const formatEmptyValue = setting => formatEmpty('value', __('Empty'), setting);
+
+const formatArraySelectionDefault = setting =>
+  formatArraySelection('default', setting);
+const formatArraySelectionValue = setting =>
+  formatArraySelection('value', setting);
+
+const formatArraySelection = (attr, setting) => {
```

Comment:
I'm missing a simmilar selector for non-grouped options :thinking:
(eg `default_locale`, `default_timezone`)